docs/contributing.md

```diff
@@ -201,87 +201,38 @@ Apache License 2.0.
 
 ## Documentation
 
-The documentation for http4s is divided into two projects: `website` and `docs`.
-
-### `website`
-
 The common area of http4s.org (i.e., directories not beginning with
-`/v#.#`) is generated from the `website` module and is published only
+`/v#.#`) is generated from the `docs` module and is published only
```

Comment:
Hmm, also confusing 😕 the module name in sbt is `site` but the directory containing the documentation is `docs/`.
```suggestion
`/v#.#`) is generated from the `docs/` directory and is published only
```